atoma-state/src/state_manager.rs

```diff
@@ -1509,23 +1518,24 @@ impl AtomaState {
         stack_small_id: i64,
         estimated_total_compute_units: i64,
         total_compute_units: i64,
-    ) -> Result<()> {
+    ) -> Result<(f64, i64)> {
         let result = sqlx::query(
             "UPDATE stacks
             SET already_computed_units = already_computed_units - ($1 - $2)
-            WHERE stack_small_id = $3",
+            WHERE stack_small_id = $3
+            RETURNING already_computed_units, 
+                CAST(already_computed_units AS FLOAT) / CAST(num_compute_units AS FLOAT)",
```

Comment:
you can change the order so you don't have to change it down

Reply:
Sure